hassio/src/addon-view/hassio-addon-dashboard.ts

```diff
@@ -250,7 +250,10 @@ class HassioAddonDashboard extends LitElement {
     }
 
     if (path === "uninstall") {
-      window.history.back();
+      // If the current URL pathname contains the addon's slug, go back to the previous page
+      if (window.location.pathname.includes(this.addon!.slug)) {
```

Comment:
instead of this check, we should do:
```ts
if (this.isConnected) {
  navigate(this._backPath);
}
```

Reply:
I was just checking on that after the CI failed. Totally agree.